lighthouse-core/report/html/renderer/util.js

```diff
@@ -65,9 +65,11 @@ class Util {
     // The proto process turns 'not-applicable' into 'not_applicable'. Correct this to support both.
     // TODO: remove when underscore/hyphen proto issue is resolved. See #6371, #6201.
     for (const audit of Object.values(clone.audits)) {
+      // TODO(exterkamp): remove this when proto enum removes 'not_applicable'
```

Comment:
remove just the `not_applicable` one? Can we add this to the proto compat umbrella bug instead of as a TODO?

Reply:
Made a tracking bug for proto convergence #6801